webapp/package.json

```diff
@@ -14,15 +14,15 @@
   },
   "devDependencies": {
     "@babel/cli": "7.16.8",
-    "@babel/core": "7.16.12",
-    "@babel/plugin-proposal-class-properties": "7.16.7",
-    "@babel/plugin-proposal-object-rest-spread": "7.16.7",
-    "@babel/plugin-proposal-optional-chaining": "7.16.7",
+    "@babel/core": "7.24.7",
+    "@babel/plugin-proposal-class-properties": "7.18.6",
+    "@babel/plugin-proposal-object-rest-spread": "7.20.7",
+    "@babel/plugin-proposal-optional-chaining": "7.21.0",
```

Comment:
I don't think we still need all these? I removed them when updating the plugin starter template https://github.com/mattermost/mattermost-plugin-starter-template/pull/213/files#diff-006cce2fd88392a0bff4cf16091d480b6238c6bdf52c9042bf7805879c791a8f

Reply:
@BenCookie95 Will look into it and let you know